.github/workflows/version-bump-publish.yml

```diff
@@ -0,0 +1,36 @@
+name: version-bump-publish
+on:
+  workflow_dispatch:
+    inputs:
+      level:
+        description: "<newversion> | major | minor | patch | premajor | preminor | prepatch | prerelease"
+        required: true
+        default: "patch"
```

Comment:
How does this work? How do I pass these args?

Reply:
You go to the Actions tab of your project, then select `version-bump-publish`, then click the `run workflow` button. You will see the following UI:

<img width="1302" alt="Screen Shot 2021-01-26 at 09 56 53" src="https://user-images.githubusercontent.com/353180/105861545-d495a780-5fbc-11eb-9e2b-c5a884b56d9a.png">

There will be some text fields where you are able to type in your options.

Reply:
Excellent :)